marketplace/deployer_util/log_util.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+
+def log(msg, *args):
```

Comment:
Can we mimic glog so that in the future we can use it as a drop-in replacement for this? This gets rid of the `ERROR` and `INFO` strings in individual log messages.

```
log.info()
log.warn()
log.error()
log.fatal()
```

(We shouldn't use glog now as its default configuration doesn't meet our need, which would require every binary to be changed to configure glog before it starts).